### ISSUE TITLE
Close sync result buffer with keybind

### ISSUE THREAD

**Issue by Thaodan** [dengste/org-caldav#286]
It would be very handy to kill the buffer similar as to similar buffer such as the `Warnings` buffer.

**Comment by jackkamm** [comment]
I agree this would be good. Probably, we should have a mode for org-caldav message and warnings buffers, that derives from `special-mode`:
https://www.gnu.org/software/emacs/manual/html_node/elisp/Basic-Major-Modes.html
That way, `q` will be bound to `quit-window`  automatically. PRs welcome...
